internal/presenters/templates/finding.component.tmpl

```diff
@@ -0,0 +1,9 @@
+{{define "finding"}}
+ ✗ [{{.Attributes.Rating.Severity.Value}}] {{.Attributes.Message.Header}}
```

Comment:
Issue: this is missing colouring and formatting. Please compare with the sarif renderer.